analytics/js_tracking_api.rst

```diff
@@ -28,10 +30,12 @@ Installing tracking code via Tag Manager
 
 Installing tracking code via code snippet.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-To install tracking code you must paste it into html file and provide some required settings:
+Installation via snippet should be done only if Tag Manager is not available or you want to track on multiple domains / subdomains.
 
- * ``$PIWIK_URL`` - location of the piwik.js file
- * ``$IDSITE`` - id of website that should be tracked
+This code should be added just before you want to use the API. Additionally snippet has to be configured this way:
+
+ * String ``//{$PIWIK_URL}/`` should be replaced with location of the piwik.js file
```

Comment:
Proposition: Use same values as in AM API instructions and snippet:

```
* String ``XXX-XXX-XXX-XXX-XXX`` should be replaces with :term:`app ID` (e.g. ``efcd98a5-335b-48b0-ab17-bf43f1c542be``).
* String ``ppms.example.com`` should be replaced with your PPMS domain name.
```

And in snippet:
```
var u="//ppms.example.com/";
_paq.push(["setTrackerUrl", u+"piwik.php"]);
_paq.push(["setSiteId", "XXX-XXX-XXX-XXX-XXX"]);
```

This should make it clear and will be consistent with other docs.